src/enroute.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-alert */
+const notification = 'You must be an admin to add car';
+
+const enroute = () => {
+  alert(notification);
```

Comment:
- [ ] Your project has 2 alerts.  Here at Microverse, alerts are highly discouraged according to the javascript [requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements):

![image](https://user-images.githubusercontent.com/57421823/212441356-e458334b-ab31-4916-9d2c-10d15b76e339.png)

You can use a modal, or a text that appears on the screen. 👍

Reply:
Changed the alert to a modal